styles/components/_logo.scss

```diff
@@ -0,0 +1,6 @@
+.starwars__logo {
+  width: 505px;
+  height: 248px;
+  background: url('../static/assets/ui/StarWarsLogo.png') center;
+  background-size: cover;
+}
```

Comment:
It's not considered a good practice to set exact px values on this element

Reply:
It is being changed when it joins the menu section.